hyperd.go

```diff
@@ -217,6 +218,17 @@ func mainDaemon(opt *Options) {
 	vmFactoryPolicy, _ := cfg.GetValue(goconfig.DEFAULT_SECTION, "VmFactoryPolicy")
 	d.Factory = factory.NewFromPolicy(d.Kernel, d.Initrd, vmFactoryPolicy)
 
+	rpcHost, _ := cfg.GetValue(goconfig.DEFAULT_SECTION, "gRPCHost")
+	rpcServer := serverrpc.NewServerRPC(d)
+	if rpcHost != "" {
```

Comment:
move `rpcServer := serverrpc.NewServerRPC(d)` into `if`
and `defer rpcServer.Stop()`

or other way avoid calling NewServerRPC()